bin/strata-cli/src/cmd/recover.rs

```diff
@@ -76,8 +79,17 @@ pub async fn recover(args: RecoverArgs, seed: Seed, settings: Settings) {
             continue;
         }
 
+        // F***! BDK 1.0 removed the `insert_tx` method, so we have to do this the hard way
```

Comment:
Is there an issue filed on this?

Reply:
No, but as said above we can use [`method.apply_unconfirmed_txs`](https://docs.rs/bdk_wallet/latest/bdk_wallet/struct.Wallet.html#method.apply_unconfirmed_txs).

Just waiting input from @Zk2u since he did a lot of development as well with BDK in Strata.

Reply:
the problem is that `apply_unconfirmed_txs` takes a `u64` timestamp as `last_seen` and that is always complicated is `async` context with tokio.